pkg/executor/docker/errors.go

```diff
@@ -0,0 +1,17 @@
+package docker
+
+import "github.com/bacalhau-project/bacalhau/pkg/models"
+
+const DOCKER_EXECUTOR_COMPONENT = "Executor/Docker"
+
+const (
+	ExectorSpecValidationErr = "ExectorSpecValidationErr"
+	ExecutionAlreadyStarted  = "ExecutionAlreadyStarted"
+	ExecutionNotFound        = "ExecutionNotFound"
```

Comment:
I think you can reuse the error codes in `/pkg/executor/types.go`

Reply:
Done.